src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { PersonRow } from './PersonRow';
+
+export const PeopleTable:
+React.FC<{ peopleFromServer: UserFromServerType[] }> = ({ peopleFromServer }) => (
```

Comment:
create a separate type Props

```suggestion
React.FC<Props> = ({ peopleFromServer }) => (
```